src/reporters/json.ts

```diff
@@ -132,9 +132,10 @@ function getMetadata() {
   };
 }
 
+/* eslint-disable-next-line @typescript-eslint/no-unused-vars */
```

Comment:
Why are we keeping this if unused? Why not use it for ECS later as we used to?

Reply:
We might reuse it once we do a PR to the playwright to expose the version.